### DISCUSSION TITLE
Cask devtunnel

### DISCUSSION THREAD

**Discussion by setaskin** [Homebrew/discussions#5532] [Casks]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
screen shot in the description
```

### Description of issue

devtunnels is in auto update list and being updated by brew bot but we got sha mismatch error for the latest release. I saw a fix PR from community but I was wondering how it was happened?  what caused this ? please see attached screen shots below. 

Brew doctor 

![image](https://github.com/user-attachments/assets/b03a0b5a-e5d4-4b66-9aa3-a74147af76f7)

@bevanjkay 

### Relevant casks

Casks/d/devtunnel.rb

**Comment by SMillerDev** [comment]
Did you retag the release?

**Reply by p-linnane** [reply]
As @SMillerDev said, this usually happens after a release is published and then later those binaries are replaced.

**Reply by klvnraju** [reply]
As far as I know, we released the binaries only once. Do not know how the bot generated different SHA.

**Reply by setaskin** [reply]
yes, as my teammate @klvnraju mentioned , we released binaries only once. and did not retag the release.  How bot generated a different one?

**Reply by bevanjkay** [reply]
The bot calculated the checksum by downloading the binaries and then generating the checksum. This was also verified in the subsequent CI runs, so this would suggest that the checksum has indeed changed.

**Reply by setaskin** [reply]
umm I don't see a change on our side, I have no idea why it happened. I will follow up in the next release, if issue continues, I will open a new discussion. We can close this one.
